testing/page-loading-spec.js

```diff
@@ -0,0 +1,28 @@
+// Copyright 2017 The TIE Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview End-to-end tests for the app.
+ */
+
+describe("Loading the homepage", function() {
+  it('should load the homepage without errors', function() {
+  browser.get('/client/app.html');
+
+  var feedbackParagraphs = element.all(by.repeater('paragraph in feedbackParagraphs track by $index'));
+  expect(feedbackParagraphs.count()).toEqual(3);
+  expect(feedbackParagraphs.get(0).getText()).toEqual('Greetings!');
+
```

Comment:
Will push another commit soon that induces console error and breaks Travis build.